README.md

```diff
@@ -46,6 +46,17 @@ All the common types used by the packages are defined [here](./common-types/inde
 
 ## Contributors
 
+### Fall 2024
+
+- **Sam Shridhar** - PM
+- **Andrew Chen** - TPM
+- **Oscar Wang** - Dev Lead/Dev Advisor
```

Comment:
I might remove this line with Oscar, since it's not officially the case (i.e. we didn't put Sarah here when she was a point of contact for IDOL). And since it's not official it's hard to explain what his role is on IDOL.